docs/fsharp/style-guide/formatting.md

```diff
@@ -1161,6 +1256,19 @@ let comparer =
               reversed.CompareTo (rev s2) }
 ```
 
+You may also prefer to use "Stroustrup" style:
+
+```fsharp
+let comparer =
+    { new IComparer<string> with
```

Comment:
Would this not be
```fsharp
let comparer = { 
    new IComparer<string> with
        member x.Compare(s1, s2) =
            let rev (s: String) = new String(Array.rev (s.ToCharArray()))
            let reversed = rev s1
            reversed.CompareTo(rev s2)
}
```
?

We don't seem to support this in Fantomas today.
If there is no reason to believe this will ever lead to invalid code, I guess we can have this as well.

Reply:
Yeah for some of these suggestions I did limit what I put in to what fantomas supports currently, but I agree this is the ideal here and I'll update it accordingly.